### PR TITLE
Fix typo 'ofters' -> 'offers'

### PR DESCRIPTION
**Pull request by jaydh**

### CODE REVIEW

File: type-definitions/Immutable.d.ts [hunk 1]
```diff
@@ -2290,7 +2290,7 @@ declare module Immutable {
    *
    * **Choosing Records vs plain JavaScript objects**
    *
-   * Records ofters a persistently immutable alternative to plain JavaScript
+   * Records offers a persistently immutable alternative to plain JavaScript
```
Comment (rgbkrk) [inline type-definitions/Immutable.d.ts]: This should probably be `offer` since Records is plural here.

Comment (rgbkrk) [reply type-definitions/Immutable.d.ts]: "Records offer a..."

**Comment by Emobe** [comment]
I already made a PR for this in #1459

**Comment by kozlitinaelja** [comment]
Thank you for working on this! I did merge #1436 which does the same thing, so I am going to close this PR

**Pull request closed without merging**